src/app.js

```diff
@@ -1,21 +1,9 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const moveFile = require('./moveFile');
 
-module.exports = sum;
+const command = process.argv[2];
+const source = process.argv[3];
+const destination = process.argv[4];
+
+moveFile(command, source, destination);
```

Comment:
It's recommended to use a try-catch block around the moveFile function call to handle any errors thrown by the function.

Reply:
Done